wiz-kubernetes-connector/values.yaml

```diff
@@ -73,26 +73,7 @@ wizApiToken:
   # API token should be read from an environment file, which is specified in podCustomEnvironmentVariablesFile
   usePodCustomEnvironmentVariablesFile: false
 
-wizConnector: # Relevant only for broker.enabled = true & autoCreateConnector = false
-  # Specifies whether a proxy secret should be created
-  # If createSecret is false you need to:
-  #  1. Create secret with this keys:
-  #     CONNECTOR_ID, CONNECTOR_TOKEN, TARGET_DOMAIN, TARGET_IP, TARGET_PORT
-  #  2. Set secretName to reference your secret
-  createSecret: true
-  # Annotations to add to the secret
-  annotations: {}
-  secretName: ""
-
-  # Required arguments for autoCreateConnector = false
-  connectorId: ""
-  connectorToken: ""
-  targetDomain: ""
-  targetIp: ""
-  targetPort: ""
-  tunnelClientAllowedDomains: []
-
-broker:
+wiz-broker:
```

Comment:
How resources/ podSecurityContext/ securityContext will be populated into the broker chart? :/
I think we need to delete all the keys that related to the broker and move it under global/ wiz-broker keys